nipype/utils/misc.py

```diff
@@ -321,21 +325,28 @@ def dict_diff(dold, dnew, indent=0):
 
     diffkeys = len(diff)
 
+    def _shorten(value):
+        if isinstance(value, str) and len(value) > 50:
+            return f"{value[:10]}...{value[-10:]}"
+        if isinstance(value, (tuple, list)) and len(value) > 10:
+            return tuple(list(value[:2]) + "..." + list(value[-2:]))
```

Comment:
I'm guessing we're never hitting this because you can't add a list and str.

```suggestion
            return tuple(list(value[:2]) + ["..."] + list(value[-2:]))
```